pkgs/development/libraries/kde-frameworks/kde-wrapper.nix

```diff
@@ -61,13 +61,9 @@ stdenv.mkDerivation {
         fi
     done
 
-    ln -s "$env/share" "$out"
-
+    mkdir -p "$out/nix-support"
     for drv in $unwrapped; do
-        if [ -a "$drv/nix-support/propagated-user-env-packages" ]; then
-            mkdir -p "$out/nix-support"
-            cat "$drv/nix-support/propagated-user-env-packages" >> "$out/nix-support/propagated-user-env-packages"
-        fi
+        echo "$drv" >> "$out/nix-support/propagated-user-env-packages"
```

Comment:
The first `buildEnv` doesn't see a collision because it hasn't included the `propagatedUserEnvPkgs` yet. The final `buildEnv` is going to look at the `propagated-user-env-pkgs` file we create in this expression and then it's going to try to include all those packages; that is when the collision will occur.